project.clj

```diff
@@ -7,7 +7,7 @@
 (def rbac-client-version "1.1.1")
 (def dropwizard-metrics-version "3.2.2")
 
-(defproject puppetlabs/clj-parent "4.9.1-SNAPSHOT"
+(defproject puppetlabs/clj-parent "4.10.0-SNAPSHOT"
```

Comment:
Does this need to be a Y if we're just releasing security bumps?

Reply:
Oh yeah, good point!

Reply:
Yeah, great point.  I'll update.